slack/src/main/java/org/togglz/slack/SlackWrapperStateRepository.java

```diff
@@ -0,0 +1,16 @@
+package org.togglz.slack;
+
+import org.togglz.core.repository.StateRepository;
+import org.togglz.core.repository.composite.CompositeStateRepository;
+
+/**
+ * @author Tomasz Skowroński
+ * @since 2.4.0
+ */
+public class SlackWrapperStateRepository extends CompositeStateRepository {
```

Comment:
I think it is a bit confusing to have `SlackWrapperStateRepository` and `SlackStateRepository`. Especially because `SlackStateRepository` doesn't really implement something that can be considered a state repository. Why not merge these into a single `SlackStateRepository` class. Then you could just pass the "real" state repository into the `SlackStateRepository` constructor and the implementation would just delegate to the real repository before or after the notification is sent. So the `SlackStateRepository` would effectively just be a wrapper that adds notification behavior to an existing repository. That's IMO much simpler to use.